test/parallel/test-event-emitter-prototype-bleeding.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+var util = require('util');
+var EventEmitter = require('events').EventEmitter;
+
+var TestClass = function () {};
+TestClass.prototype = new EventEmitter;
+
+function listener_n1() {
+	// This one is okay to be called!
```

Comment:
This function must be called, right? Why not assign some value to a variable here and check if the value is set in `process.exit`?